ultimate_hosts_blacklist/whitelist/__init__.py

```diff
@@ -277,14 +278,16 @@ def _command_line():
         "--whitelist",
         type=argparse.FileType("r"),
         nargs="+",
-        help="Read the given file and append its data to the our whitelist list.",
+        help="Read the given file and append its data to the UHBW's\n"
+        "hosted whitelist list.",
     )
 
     parser.add_argument(
-        "-wc",
+        "-wc\n",
         "--without-core",
         action="store_false",
-        help="Disable the usage of the Ultimate Hosts Blacklist whitelist list.",
+        help="Disable the usage of the Ultimate Hosts Blacklist\n"
```

Comment:
If linebreak is also needed from the CLI when you call `--help`, you need to implement what I did inside PyFunceble: 
 https://github.com/funilrys/PyFunceble/blob/d6033601909e88495f5b102e45ee7d5a48dbe46a/PyFunceble/cli/__init__.py#L122

Reply:
I leave this implementation to you @funilrys as I would not try to looks like I know anything about python :smile: which I don't